docs/api/addons.md

```diff
@@ -0,0 +1,275 @@
+---
+title: 'Addons'
+---
+
+Addons extend Storybook with features and integrations that are not built into the core. Most Storybook features are implemented as addons. For instance: documentation, accessibility testing, interactive controls, and design previews. 
+Storybook’s addon API makes it easy for you to configure and customize Storybook in new ways. There are countless addons made by the community that unlock time-saving workflows.  What addons can do:
+
+- [Add a panel to Storybook (like Action Logger).](../essentials/actions)
+- [Add a tool to Storybook’s toolbar (like zoom or grid).](../essentials/toolbars-and-globals)
+- [Add a tab to Storybook (like SB Docs).](../writing-docs/introduction)
+
+Browse the [Addon gallery](/addons) to install an existing addon or as inspiration for your own addon. Read on to learn how to make an addon yourself.
+
+### Storybook basics
+
+Before writing your first addon, let’s take a look at the basics of Storybook’s architecture. While Storybook presents a unified user interface, under the hood it’s divided down the middle into **Manager** and **Preview**.
+
+The Manager is the UI where Storybook’s search, navigation, toolbars, and addons are rendered. The Preview area is an iframe where stories are rendered.
+
+![Storybook detailed window](./manager-preview.jpg)
+
+Because Manager and Preview run in separate iframes, they communicate across a communication channel. For example, when you select a story within the Manager, an event is sent across the channel, and the selected story is rendered inside the Preview.
```

Comment:
```suggestion
Because Manager and Preview run in separate iframes, they communicate across a communication channel. When you select a story within the Manager an event is sent across the channel and the selected story is rendered inside the Preview.
```

Reply:
Fixed with latest commit